### PR TITLE
Next major

### PR DESCRIPTION
**Pull request by greg0ire**
Add missing next_major tags

### CODE REVIEW

File: Tests/Util/AdminObjectAclDataTest.php [hunk 1]
```diff
@@ -92,8 +90,6 @@ public function testSetForm()
      * @depends testSetForm
      *
      * @group legacy
-     *
-     * @deprecated
```
Comment (core23) [inline Tests/Util/AdminObjectAclDataTest.php]: Shouldn't we add `NEXT_MAJOR` here too?

Comment (greg0ire) [inline Tests/Util/AdminObjectAclDataTest.php]: I'm not sure: we will remove the code under test, which will make us notice this.

**Comment by OskarStark** [comment]
@greg0ire can you change the commit message accordingly to the others i recently merged?

thank you 👍

**Comment by greg0ire** [comment]
@OskarStark : I don't get it, the message is also "Add missing NEXT_MAJOR tags" here.

**Comment by OskarStark** [comment]
> @OskarStark : I don't get it, the message is also "Add missing NEXT_MAJOR tags" here.

sorry ypu are right, i just saw the PR title... sunday... 😄

**Comment by OskarStark** [comment]
anything todo here @greg0ire @core23 ?

**Comment by greg0ire** [comment]
Nope, good to go IMO

**Comment by OskarStark** [comment]
Thank you @greg0ire 👍

**Pull request merged**